tox.ini

```diff
@@ -0,0 +1,9 @@
+[tox]
+envlist = py27,py33
```

Comment:
should align this with travis-ci, which uses 3.4 and 3.5, no 3.3.  i looked briefly at tox docs and didn't see a `py35` entry?